### PR TITLE
Add Swagger documentation

### PR DESCRIPTION
**Pull request by AndreyWolox**
## Summary
- add  swagger dependence
- add swagger documentation
- add swagger config
## Trello Card
https://trello.com/c/nyr7oX0D/40-swagger

### CODE REVIEW

File: src/main/java/wolox/training/controllers/BookController.java [hunk 1]
```diff
@@ -79,6 +84,8 @@ public Book findOne(@PathVariable Long id) {
      */
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
+    @ApiOperation(value = "Create a book", response = Book.class)
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "Successfully created Book")})
```
Comment (leandrocostawolox) [inline src/main/java/wolox/training/controllers/BookController.java]: 201

File: src/main/java/wolox/training/models/Book.java [hunk 2]
```diff
@@ -10,13 +12,16 @@
...
+@ApiModel(description = "Model to the Book")
 public class Book {
 
     @Id
     @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @ApiModelProperty(value = "The identifier of the book", example = "321")
     private long id;
 
     @NotNull
+    @ApiModelProperty(value = "The gener of the book", example = "Action")
```
Comment (leandrocostawolox) [inline src/main/java/wolox/training/models/Book.java:24]: *genre

**Review by leandrocostawolox — COMMENTED** [review]
Two minor comments 👼

File: src/main/java/wolox/training/models/Book.java (hunk 2 above)
Comment (leandrocostawolox) [inline src/main/java/wolox/training/models/Book.java]: *genre

**Review by leandrocostawolox — CHANGES_REQUESTED** [review]

**Review by CarlosLugo22 — APPROVED** [review]
Great, the comments made by Lea was worked, Good

**Review by leandrocostawolox — APPROVED** [review]

**Pull request merged**